deployment/debian-package-arm64/docker-build.sh

```diff
@@ -11,6 +11,21 @@ pushd ${SOURCE_DIR}
 # Remove build-dep for dotnet-sdk-2.2, since it's not a package in this image
 sed -i '/dotnet-sdk-2.2,/d' debian/control
 
+# Clone down and build Web frontend
```

Comment:
this is going to be copy-pasted all over the place, can it be shared in some common `.sh`?

Reply:
I hate doing that because it makes each script unreadable on its own with calls out to random other places.

Reply:
Especially since this is a script called within a dozen different Docker containers. Trying to separate this out is just going to be unreadable.